mkdocs.yml

```diff
@@ -170,14 +176,8 @@ pages:
           - Configuration: agents/sematext-agent/packages/configuration.md
           - Data: agents/sematext-agent/packages/data.md
         - Auto Discovery:
-          - Removing stale resources: agents/sematext-agent/autodisco/removing-stale-resources.md 
+          - Removing stale resources: agents/sematext-agent/autodisco/removing-stale-resources.md
           - JMX in Container Environments: agents/sematext-agent/jmx-attaching-containers.md
-      - Infra & App Agent:
```

Comment:
We need to add redirects for all files/pages we've moved or deleted.  Please ask Ciprian to add those redirects.